Source/Particles/Filter/FilterFunctors.H

```diff
@@ -100,9 +102,15 @@ struct ParserFilter
         amrex::Real const x  = p.pos(0);
         amrex::Real const y  = p.pos(1);
         amrex::Real const z  = p.pos(2);
-        amrex::Real const ux = p.rdata(PIdx::ux)/PhysConst::c;
-        amrex::Real const uy = p.rdata(PIdx::uy)/PhysConst::c;
-        amrex::Real const uz = p.rdata(PIdx::uz)/PhysConst::c;
+        amrex::Real ux = p.rdata(PIdx::ux)/PhysConst::c;
```

Comment:
I find this a bit confusing, once its in SI, once in SI/c (so basically relativistic mass) and once in beta*gamma (below).
When do we use the 2nd case?

Reply:
Ah, now I get it :)
Input is proper velocity: Before its in gamma*v, now it's in beta * gamma.
Input is SI: before it's gamma m v, now it's beta * gamma.